charts/kube-prometheus-stack/templates/prometheus/rules-1.14/k8s.rules.container_memory_cache.yaml

```diff
@@ -32,11 +32,15 @@ spec:
       record: node_namespace_pod_container:container_memory_cache
       {{- if or .Values.defaultRules.additionalRuleLabels .Values.defaultRules.additionalRuleGroupLabels.k8sContainerMemoryCache }}
       labels:
-        {{- with .Values.defaultRules.additionalRuleLabels }}
-          {{- toYaml . | nindent 8 }}
+        {{- range $k, $v := mergeOverwrite
+              (.Values.defaultRules.additionalRuleLabels | default (dict))
+              (.Values.defaultRules.additionalRuleGroupLabels.k8sContainerMemoryCache | default (dict))
+            }}
+
+        {{- if $v }}
```

Comment:
Thats not a great condition. It prevent label values like 0 or false.

Reply:
You're right, thank you I'll fix this up